### ISSUE TITLE
[CAIA a11y Research - PILOT]: IIR, Welcome Pathway (Desktop) (Nov. 2024)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#95525]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled` 
> - **PILOT DATE:** 11/27/2024 at 9:00 - 10:30 a.m. EST
> - **TYPES OF ASSISTIVE TECH:**  JAWS to be used in pilot
>    - **Study AT:** _Desktop Screen Reader, Mobile Screen reader, Magnification/Zoom, Speech input Tech (Sire, Dragon)_ 
> - **TESTING ENVIRONMENT:** [Codespace] (🔗 `Pending Dev.`)
> - **RESEARCH MATERIALS:** [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20research%20plan.md), [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20conversation%20guide.md) & [Research Goals](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/research%20plan%202024.md#research-goals)

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- **Pilot**: @SarahKay8 
- **Co-pilot**: @EvanAtCoforma 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: IIR (Iterate, Innovate, Run)
- OCTO-DE product owner: Jeff Barns
- Product manager: Pete Egan
- Designer: Liza McRuer/Raquel Eisele
- FE engineer: John Rodriguez/Nick Sayre
- Product/team Slack channel: #iir-product-teams-public

</details>

## Research Goals

<details><summary>Toggle to view team research goals</summary>

**Review extracted from the [Fall 2024 Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20research%20plan.md#research-goals) found below:** 

> ### What are you trying to learn from this research?
> - Identify if we are meeting the VA’s experience standards of comprehension, consistency, credibility, efficiency, findability, health and safety, and usability are met for Veterans using Assistive Technology (AT).
> - Determine if Veterans using AT find the proposed alert and update/edit contact information designs simple to navigate and and the information easy to update.
> 
> ### Outcome
> 
> #### How will this research advance your product to the next phase in the design process? What will you do with your learnings?
> 
> - Our learnings will inform us of changes that needs to be made including:
> - Adjustments to align with VA experience standards
> - Updates to the Welcome message to support Assistive Technology(AT)
> - Updates to the contact information experience to support Assistive Technology(AT) 
> **We will make changes as we can to the designs to support the use of AT.** 
> 
> ### Research questions
> 
> - Are participants able to successfully navigate to key tasks on the website using assistive technology (keyboard, voice, and zoom/magnifier)?
>     - Which tasks, if any, do participants struggle finding? Where do they expect to find those tasks?
>     - Which tasks, if any, do participants struggle completing? How do they expect to complete those tasks?
> 
> ### Hypothesis
> 
> **Based on the previous research, we hypothesize that we’ll confirm that:**
> 1. Focused actions that will improve their overall experience. Veterans reported that the “complexity of VA discourages people from applying for benefits, especially for people with cognitive considerations” (TSM: Research Review). While we cannot simplify eligibility requirements and application process within the scope of this project, we can reduce the cognitive load of account creation by providing quick actions, like reviewing contact information, that will support the Veterans' overall experience when they do begin to apply for benefits.
> 2. We also hypothesize that participants will be able to complete tasks proposed to them and will have a clear understanding of the value the proposed tasks provide.

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:**  Week of 11/25/2024 on 11/27/2024
- **Access prototype on staging using this test account**: 
[Link to VA staging](https://staging.va.gov/) 
test account username login: vets.gov.user+1@gmail.com
test account password: DbAxDhMy5L084
- [Link to Figma prototype](https://www.figma.com/proto/CertVj7cu66kFv9TnseR30/Veteran-Onboarding?page-id=300%3A11104&node-id=300-11287&viewport=553%2C142%2C0.13&t=K7BD3b37FTWP1gKA-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=300%3A11287&show-proto-sidebar=1)
- [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20research%20plan.md)
- [Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20conversation%20guide.md) 
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `a11y` will test using `AT` on `Browser` and `Device`
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @NaomiPMC  in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11ys** 11/27/2024
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] HOLD placed the week of 11/11/2024 on CAIA Calendar
- [x] Holds updated on 11/13/2024 for week of 11/25/2024
- [x] **Determine date and time for pilot** 11/27/2024 at 9 a.m. EST
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Receive Research Goals** add to ticket section above for goals.
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20conversation%20guide.md)
- [x] Request AT device and tech to be used in sessions 
- [x] Receive AT device and tech from Team in research ticket `JAWS` to prep for pilot
- [ ] **Review prototype** in `Codespace`
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** [Notes doc](https://docs.google.com/document/d/13vpvKBDn0Ayho0s0jpGZedpYdtmavLdLFzdfo2FVxZY/edit?usp=sharing)
- [ ] Share findings with team from Pilot doc.
```

**Comment by sara-amanda** [comment]
## Timeframe Shift 
- 2 weeks until pilots
- Week of November 25 for pilots (TBD)
- Will update Capacity Calendar Accordingly

### Slack Update
`iir-product-teams-public`

> Raquel Eisele [Yesterday at 5:28 PM](https://dsva.slack.com/archives/C05RJS5DANT/p1731450522330639?thread_ts=1731425468.647049)
> Hey Jamie, I just had a conversation with my dev and it sounds like sadly we are not ready to conduct the pilot this week. If we implement the changes you highlighted we believe we can be ready for our pilot in two weeks and if we don't we believe we will be ready by next week. Can we adjust our plans with your team to accommodate that change?

`sitewide-content-accessibility-ia`

> Raquel Eisele
[Thursday Nov. 7 at 1:53 PM](https://dsva.slack.com/archives/C01K37HRUAH/p1731005598123579?thread_ts=1730990782.339339&cid=C01K37HRUAH)
>
> - Thank you so much for your response 
> - Sara (https://dsva.slack.com/team/U04NBMUH5C4)
> - **In response to your questions:** 
>     - **When are you looking to hold the Pilots?** 
>         - We would like to hold them later next week or early the week of the 18th. 
>         - I did start talking to Jamie about this.
>     - **What AT are you interested in testing?** 
>         - Desktop Screen Reader, Mobile Screen reader, Magnification/Zoom and Speech input Tech (Sire, Dragon).
>    - **What devices are you interested in testing?**
>         - We are interested in testing both mobile and desktop.
> - Thank you for sharing the Assistive Tech Pilot Guide!

cc: @coforma-jamie

**Comment by raquelou** [comment]
We have run into some issues with the test user accounts. We will reach out with updates as they occur.

**Comment by coforma-jamie** [comment]
[Pilot notes & findings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95851#issuecomment-2512343182)

**Comment by coforma-jamie** [comment]
This ticket is ready for closure review

**Comment by SarahKay8** [comment]
HI @raquelou! Here are my findings for the Welcome page modal - after signing out with the test user, I wasn't able to repopulate the modal message and was unable view the code in Devtools. I am just writing down recommendations based on the screen reader output I received 

### Welcome Modal Audit Findings - NVDA + Voiceover (Desktop)
Test User - adam.king0126+1@gmail.com P: 369SsNrLgPv5

**NVDA Output** 
Redundant Output
	**Issue**
      The Close button is announced twice 
	•	“Close welcome to my VA modal button.”
	•	“Button close welcome to my VA modal.”
	This repetition might confuse screen reader users and could lead to unnecessary cognitive load.
	**Recommendation:**
	•	Ensure button has only one accessible name, and remove redundancy. Use proper labeling with aria-label or a clear button name: `<button aria-label="Close Welcome to My VA modal">Close</button>`

**Loading Progress Bar Announcement**
	•	Issue: The “Loading your information. Loading progress bar” is announced before the modal content. While it provides useful information about loading, it interrupts the user’s navigation to the modal content.
	•	Recommendation:
	•	If the progress bar is no longer relevant when the modal loads, hide it from screen readers using 
aria-hidden="true":
`<div aria-hidden="true">Loading progress bar</div>
`

List Structure for Continue Button 
       NVDA mentions “List with one item” for the Continue button, which is unnecessary because there is only one item.
	•	Recommendation:
	      Make sure the button isn't wrapped in an unnecessary list` (< ul > or < ol >)`. If it’s part of a single-item list for styling purposes, use role="none" or aria-hidden="true" on the list to hide it from screen readers:
```
< ul role =" none " >
  < li >
    < button > Continue < / button >  

  < /li >
< /ul >

```

### Voiceover Output 

**Continue Button misidentification** 

The “Continue” button is being announced as part of a group instead of being identified as a standalone button. This could confuse users relying on screen readers.
	
**Recommendation:** 
Make sure the “Continue” button is marked as a button 

If the button is nested within an unnecessary group or wrapper, VoiceOver might announce it as a “group” instead of a clear interactive element.
	•	Simplify the DOM structure and remove unnecessary wrapper elements around the button.
	•	If a group is necessary (e.g., for layout), apply aria-hidden="true" to hide the wrapper from screen readers:
```
<div aria-hidden="true">
  <button id="continue">Continue</button>
</div>

```
